schema/attestation.graphqls

```diff
@@ -0,0 +1,39 @@
+scalar JSON
+
+extend type Query {
+  """
+  attestations returns all attestations for a given vehicle token.
+
+  Required Privileges: [ATTESTATION_CREDENTIAL]
+  """
+  attestations(
+    """
+    The token ID of the vehicle.
+    """
+    tokenId: Int!
+    """
+    The attesting party. 
+    """
+    signer: String
+  ): [Attestation]
```

Comment:
Should we worry about pagination here? This could be really large.

Reply:
I agree this could get very large.. I'm going to punt on this for now just to see this working on dev and will make pagination a fast follow!